Codeinwp/neve
# Add global color as presets in beaver builder

### Summary
- Added Neve global colors to Beaver Builder color presets.
Beaver is only working with hex values so if you set, let's say, title color to have the primary color when you change the color in the customizer, the color in Beaver that you just set won't change, it will just register the new color in Beaver presets ( if it doesn't already exist )

### Will affect visual aspect of the product
NO

### Screenshots 
- https://prnt.sc/11r02ee
- https://prnt.sc/11r04lo

### Test instructions
- Edit a page in beaver builder
- Add an icon widget and try to set its color
- See if the colors declared in the customizer are visible on presets too
- Choose a color
- Go to the customizer and change a global color or the active palette
- All the colors from the previous palette and the new colors from the new palette should be there
- Try to add a custom color preset, save the page, and see if it remained after the refresh https://prnt.sc/11r0d56
- Check with rgba colors too



Closes #2328.
